experiment/coverage/apicoverage.go

```diff
@@ -100,15 +100,15 @@ func getOpenAPISpec(url string) apiArray {
 }
 
 //   I0919 15:34:14.943642    6611 round_trippers.go:414] GET https://172.27.138.63:6443/api/v1/namespaces/kube-system/replicationcontrollers
-var reAPILog = regexp.MustCompile(`round_trippers.go:\d+\] (GET|PUT|POST|DELETE|OPTIONS|HEAD|PATCH) (\S+)`)
+var reE2eAPILog = regexp.MustCompile(`round_trippers.go:\d+\] (GET|PUT|POST|DELETE|OPTIONS|HEAD|PATCH) (\S+)`)
```

Comment:
Cool, thanks for your advice.
OK, will do that.

Reply:
I tired to implement this point, but I feel it is better to avoid it with 2 reasons.
1. On apiserver log, we need to remove ":" at the end of API URI because the log like
```
1 wrap.go:42] PUT /apis/apiregistration.k8s.io/v1/apiservices/v1.apps/status:
```
So maybe the shared method will be a little mess if we merge them into a single method.
2. We will want to add client analyzer (curl/7.58.0, dashboard/v1.8.3, e2e.test/v0.0.0, etc) into apiserver parser. So it is better to keep the methods separated for adding that.

Reply:
Ahh yes if we add client analysis and the syntax isn't identical, I agree. Let's keep them separate.

Reply:
@stevekuznetsov 
Then can I get your +1 on this PR ?